src/cmd/ksh93/data/signals.c

```diff
@@ -107,6 +107,9 @@ const struct shtable2 shtab_signals[] =
 #else
 	"INT",		VAL(SIGINT,SH_SIGINTERACTIVE),			"",
 #endif /* JOBS */
+#ifdef SIGINFO
+	"INFO",		VAL(SIGINFO,SH_SIGDONE),			S("Information request"),
```

Comment:
I'm pretty sure that `SIGINFO` is not meant to terminate the process by default, so the `SH_SIGDONE` default action should probably be `SH_SIGIGNORE` (it was added this way in ksh2020 as well). See: https://github.com/ksh93/ksh/blob/d7afb57c49803c2194ccad68a34e6827f20e76b5/src/cmd/ksh93/include/fault.h#L61-L62

Reply:
That was a copy paste error from the `SIGIO` entry. `SH_SIGIGNORE` is now used instead.